### ISSUE TITLE
[Python 3.9] AttributeError: 'array.array' object has no attribute 'tostring'

### ISSUE THREAD

**Issue by rbn42** [python-xlib/python-xlib#184]
I found this error after I updated my ArchLinux to python 3.9
```
  File "my_own_script.py", line 28, in get_wm_class_and_state
    wm_state = win.get_wm_state()
  File "/usr/lib/python3.9/site-packages/Xlib/xobject/drawable.py", line 765, in get_wm_state
    return self._get_struct_prop(atom, atom, icccm.WMState)
  File "/usr/lib/python3.9/site-packages/Xlib/xobject/drawable.py", line 782, in _get_struct_prop
    value = r.value.tostring()
AttributeError: 'array.array' object has no attribute 'tostring'
```

**Comment by petli** [comment]
This is fixed in 0.29, try upgrading to it: https://github.com/python-xlib/python-xlib/releases/tag/0.29

**Comment by rbn42** [comment]
Thank you.
